spynnaker_integration_tests/test_external_devices/test_live_neuron_voltage.py

```diff
@@ -133,11 +133,10 @@ def live_neuron_voltage():
     assert(numpy.sum(translator_2.voltages[key_2_1]) == 0)
     assert(numpy.sum(translator_2.voltages[key_2_2]) == 0)
 
-    i = 0
+    # We can get packet loss so don't assume all voltages exist
```

Comment:
This comments say you can loose packets but code expects all there just reordered.

Reply:
It expects all the ones that are received to actually be recorded; those can't go missing.